CONTRIBUTING.md

```diff
@@ -1,270 +1,59 @@
-# Contributing to Hack Club
+# How To Contribute
 
-Contributions to Hack Club are very welcome, and strongly encouraged (even to
-this document)!
+This is a guide explaining the contribution workflow we use at Hack Club!
 
-## Doing it
+- [Find something you want to change](#find-something-to-change)
+- [If it is big, talk about it on Slack](#talk-about-it-on-slack)
+- [Create an issue if you want to proceed](#create-an-issue)
+- [Make a PR](#make-a-pr)
+- [Wait for review](#wait-for-review)
 
-We use a modified version of
-[GitHub Flow](https://guides.github.com/introduction/flow/) at Hack Club. The
-only difference is instead of deploying from a reviewed pull request, we merge
-first and deploy straight from master.
+## Find Something To Change
 
-Part of the GitHub Flow is submitting pull requests. See
-https://help.github.com/articles/using-pull-requests/ for a good overview of
-what pull requests are and how to use them. Internally, we use the _shared
-repository model_ for collaborative development, though we encourage external
-contributors to contribute through the _fork & pull model_.
+Usually, as a contributor you know what you want to work on. Perhaps something
+along the lines of workshops, standards or implementing guidelines. However, if
+you do not know where to begin [you can always search through the open GitHub
+issues](https://github.com/hackclub/hackclub/issues) to find something suitable.
+If you are a complete beginner (or have never contributed to an Open Source
+repository before) you may be interested in [checking out issues with the
+`first-timers-only` tag](https://github.com/hackclub/hackclub/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Afirst-timers-only).
 
-When contributing, please pay attention to the commit format of other commits in
-the repository.
+## Talk About It On Slack
 
-### Style Guides
+Wasting time is not fun! If you are going to make a big change to a workshop
+(or even adding a new one) it is recommended that you
+[join our Slack](https://slack.hackclub.io) and talk
```

Comment:
I would split this up into a bulleted point list like [here](https://github.com/hackclub/hackclub/blob/master/workshops/CONTRIBUTING.md#we-love-questions) because ordered lists of steps are easier to understand.